recipes-core/images/mistysom-image.bbappend

```diff
@@ -1,4 +1,4 @@
-CORE_IMAGE_BASE_INSTALL += "pv kernel-modules weston weston-init weston-examples gtk+3-demo clutter-1.0-examples"
+CORE_IMAGE_BASE_INSTALL += "pv htop kernel-modules weston weston-init weston-examples gtk+3-demo clutter-1.0-examples"
```

Comment:
Do we really need `htop`? Or is it "just" added junk we don't need? If there's no good reason to add it, I vote for removing it

Reply:
It makes my life easier. I am not comfortable working with `top`.

Reply:
You're twisting my arm! 
Since it's a relatively small utility (~346KB),  we can live with it.